examples/01_basic/modal_acoustics_analysis.py

```diff
@@ -83,478 +163,561 @@ def display_image(image_name):
-    os.path.join(cwd, "geometry.png"), image_export_format, settings_720p
+# Create a selection info object for the geometry entities
+fvert = sel_manager.CreateSelectionInfo(SelectionTypeEnum.GeometryEntities)
+# Set the list of vertices as the geometry entities
+fvert.Entities = vertices
+
+# Add a fixed support to the modal acoustic analysis
+fixed_support = modal_acst.AddFixedSupport()
+# Set the location of the fixed support to the geometry entities
+fixed_support.Location = fvert
+
+# Activate the modal acoustic analysis and display the image
+modal_acst.Activate()
+set_camera_and_display_image(
+    camera, graphics, settings_720p, output_path, "geometry.png"
 )
-display_image("geometry.png")
 
 # %%
-# Add results
-# ~~~~~~~~~~~
-# Add 10 modes
+# Add results to the solution
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+# Get the solution object from the modal acoustic analysis
+solution = model.Analyses[0].Solution
 
-soln = Model.Analyses[0].Solution
-TOT_DEF1 = soln.AddTotalDeformation()
-TOT_DEF2 = soln.AddTotalDeformation()
-TOT_DEF2.Mode = 2
-TOT_DEF3 = soln.AddTotalDeformation()
-TOT_DEF3.Mode = 3
-TOT_DEF4 = soln.AddTotalDeformation()
-TOT_DEF4.Mode = 4
-TOT_DEF5 = soln.AddTotalDeformation()
-TOT_DEF5.Mode = 5
-TOT_DEF6 = soln.AddTotalDeformation()
-TOT_DEF6.Mode = 6
-TOT_DEF7 = soln.AddTotalDeformation()
-TOT_DEF7.Mode = 7
-TOT_DEF8 = soln.AddTotalDeformation()
-TOT_DEF8.Mode = 8
-TOT_DEF9 = soln.AddTotalDeformation()
-TOT_DEF9.Mode = 9
-TOT_DEF10 = soln.AddTotalDeformation()
-TOT_DEF10.Mode = 10
+# Add 10 total deformation results, setting the mode for every result except the first
+total_deformation_results = []
+for mode in range(1, 11):
+    total_deformation = solution.AddTotalDeformation()
+    if mode > 1:
+        total_deformation.Mode = mode
+    total_deformation_results.append(total_deformation)
 
 # %%
-# Add acoustic pressure
+# Add the acoustic pressure result to the solution
 
-ACOUST_PRES_RES = soln.AddAcousticPressureResult()
+acoustic_pressure_result = solution.AddAcousticPressureResult()
 
 # %%
-# Add force reaction scoped to fixed Support
+# Scope the force reaction to the fixed support
 
-FORCE_REACT1 = soln.AddForceReaction()
-FORCE_REACT1.BoundaryConditionSelection = FIXED_SUPPORT
+# Add a force reaction to the solution
+force_reaction_1 = solution.AddForceReaction()
+# Set the boundary condition selection to the fixed support
+force_reaction_1.BoundaryConditionSelection = fixed_support
 
 # %%
-# Solve
-# ~~~~~
+# Solve the solution
+# ~~~~~~~~~~~~~~~~~~
 
-soln.Solve(True)
+solution.Solve(True)
 
 # sphinx_gallery_start_ignore
-assert str(soln.Status) == "Done", "Solution status is not 'Done'"
+assert solution.Status == SolutionStatusType.Done, "Solution status is not 'Done'"
 # sphinx_gallery_end_ignore
 
 
 # %%
-# Messages
-# ~~~~~~~~
+# Print messages
+# ~~~~~~~~~~~~~~
 
-Messages = ExtAPI.Application.Messages
-if Messages:
-    for message in Messages:
+messages = app.ExtAPI.Application.Messages
+if messages:
```

Comment:
There's already a `show()` method on the message class. I think we should use it rather than define our own in these examples.